composer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "civicrm/civicrm-packages",
+  "description": "Legacy third party dependencies for CiviCRM",
+  "type": "library"
```

Comment:
@dsnopek will we want a mention of composer-installers here given we will need to customise it goes in the civicrm/ folder within library and also shouldn't we have require: { civicrm/civicrm: xxx } ?